tests/functional/test_endpoints.py

```diff
@@ -92,15 +152,22 @@ def test_service_name_matches_endpoint_prefix():
 
 
 def _assert_service_name_matches_endpoint_prefix(loader, service_name):
-    # Load the service model and grab its endpoint prefix
     service_model = loader.load_service_model(service_name, 'service-2')
-    endpoint_prefix = service_model['metadata']['endpointPrefix']
+    computed_name = _get_computed_service_name(
+        service_name, service_model['metadata'])
```

Comment:
I added `service_id` property the other [day](https://github.com/boto/botocore/pull/1467). I think we can just use that. It will also raise errors if it is not present.